components/layout_2020/flow/inline.rs

```diff
@@ -125,6 +127,34 @@ impl<'box_tree, 'a, 'b> InlineFormattingContextState<'box_tree, 'a, 'b> {
 
         self.positioning_context.push(hoisted_box);
     }
+
+    /// Finish layout of all the partial inline boxes in the current line,
+    /// finish current line and start a new one.
+    fn refresh_newline(&mut self, layout_context: &LayoutContext) {
```

Comment:
This could probably be called `finish_line_and_reset_inline_positions` just to be a bit more obvious what it does and to make it match the naming of `finish_line()`.

Reply:
How about `finish_line_reset` ?